contacts-app/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
where are the used dependencies like mongojs did you forget to save your dependencies in package.json